tests/drivers/gpio/gpio_basic_api/src/main.c

```diff
@@ -81,13 +81,6 @@ static void board_setup(void)
 			    IOMUXC_SW_PAD_CTL_PAD_PKE_MASK |
```

Comment:
not related to this PR, but since I see it: @danieldegrasse can you review if this is still needed?